### PR TITLE
add troubleshooting.mdx

### PR DESCRIPTION
**Pull request by RazcoDev**

### CODE REVIEW

File: docs/embeddable-uis/troubleshooting.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+In this page, we will cover some common issues that you might encounter while embedding Permit Elements in your application.
+
+:::tip
+In most cases, we recommend using **"Developer Tools" -> "Network"** (tab) in your browser to debug the requests and responses of the login method or `/me` requests.
+![Network tab](/img/elements/troubleshooting/devtools.png)
+:::
+
+## General Issues
+Before diving into the specific issues, here are some general issues that you might encounter while embedding Permit Elements in your application.
+- Make sure the Element's URL you set in the `<iframe>` tag is correct - Use the "Generate Code" button from the Element's configuration page.
```
Comment (obsd) [inline docs/embeddable-uis/troubleshooting.mdx:15]: Check if you can add link here

File: docs/embeddable-uis/troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+- Make sure you implement the login method correctly both in your **frontend** and **server** applications - follow the [Login Methods] guide to make sure that the login method is implemented correctly.
```
Comment (obsd) [inline docs/embeddable-uis/troubleshooting.mdx:16]: Add link to the login methods in the regular guide

File: docs/embeddable-uis/troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+- Make sure that the server is running and accessible from the frontend application - check the server logs for more information.
```
Comment (obsd) [inline docs/embeddable-uis/troubleshooting.mdx:17]: or the network tab in the dev tools

File: docs/embeddable-uis/troubleshooting.mdx (hunk 1 above)
Comment (RazcoDev) [reply docs/embeddable-uis/troubleshooting.mdx:15]: It's quite hard to add a link here to your specific element :/

File: docs/embeddable-uis/troubleshooting.mdx (hunk 3 above)
Comment (RazcoDev) [reply docs/embeddable-uis/troubleshooting.mdx:17]: added

**Review by maya-barak — APPROVED** [review]
STAMP

**Pull request merged**